filament/backend/src/webgpu/WebGPUDriver.cpp

```diff
@@ -123,6 +349,198 @@ void WebGPUDriver::destroyDescriptorSetLayout(Handle<HwDescriptorSetLayout> tqh)
+}
+
+Handle<HwFence> WebGPUDriver::createFenceS() noexcept {
+    return Handle<HwFence>((Handle<HwFence>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwTimerQuery> WebGPUDriver::createTimerQueryS() noexcept {
+    return Handle<HwTimerQuery>((Handle<HwTimerQuery>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwIndexBuffer> WebGPUDriver::createIndexBufferS() noexcept {
+    return Handle<HwIndexBuffer>((Handle<HwIndexBuffer>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwTexture> WebGPUDriver::createTextureViewS() noexcept {
+    return Handle<HwTexture>((Handle<HwTexture>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwBufferObject> WebGPUDriver::createBufferObjectS() noexcept {
+    return Handle<HwBufferObject>((Handle<HwBufferObject>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwRenderTarget> WebGPUDriver::createRenderTargetS() noexcept {
+    return Handle<HwRenderTarget>((Handle<HwRenderTarget>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwVertexBuffer> WebGPUDriver::createVertexBufferS() noexcept {
+    return Handle<HwVertexBuffer>((Handle<HwVertexBuffer>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwDescriptorSet> WebGPUDriver::createDescriptorSetS() noexcept {
+    return Handle<HwDescriptorSet>((Handle<HwDescriptorSet>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwRenderPrimitive> WebGPUDriver::createRenderPrimitiveS() noexcept {
+    return Handle<HwRenderPrimitive>((Handle<HwRenderPrimitive>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwVertexBufferInfo> WebGPUDriver::createVertexBufferInfoS() noexcept {
+    return Handle<HwVertexBufferInfo>((Handle<HwVertexBufferInfo>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwTexture> WebGPUDriver::createTextureViewSwizzleS() noexcept {
+    return Handle<HwTexture>((Handle<HwTexture>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwRenderTarget> WebGPUDriver::createDefaultRenderTargetS() noexcept {
+    return Handle<HwRenderTarget>((Handle<HwRenderTarget>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwDescriptorSetLayout> WebGPUDriver::createDescriptorSetLayoutS() noexcept {
+    return Handle<HwDescriptorSetLayout>(
+            (Handle<HwDescriptorSetLayout>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwTexture> WebGPUDriver::createTextureExternalImageS() noexcept {
+    return Handle<HwTexture>((Handle<HwTexture>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwTexture> WebGPUDriver::createTextureExternalImage2S() noexcept {
+    return Handle<HwTexture>((Handle<HwTexture>::HandleId) nextFakeHandle++);
+}
+
+Handle<HwTexture> WebGPUDriver::createTextureExternalImagePlaneS() noexcept {
+    return Handle<HwTexture>((Handle<HwTexture>::HandleId) nextFakeHandle++);
+}
+
+void WebGPUDriver::createSwapChainR(Handle<HwSwapChain> sch, void* nativeWindow, uint64_t flags) {
+    WebGPUPlatform::SurfaceBundle surfaceBundle = mPlatform.createSurface(nativeWindow, flags);
+    mSurface = std::move(surfaceBundle.surface);
+    // TODO we are ignoring surfaceBundle.fallbackExtent for now
```

Comment:
`fallbackExtent` was a thing only for wayland IIRC. It's probably ok to remove it for now, and then add something similar if ever needed.